dotnet/src/Extensions/Extensions.UnitTests/TemplateEngine/Prompt/PromptTemplateEngineTests.cs

```diff
@@ -75,18 +78,24 @@ public void ItRendersVariables()
         Assert.Equal(BlockTypes.Code, blocks[8].Type);
         Assert.Equal(BlockTypes.Code, updatedBlocks[8].Type);
 
+        Assert.Equal("food a='b' c = $d", blocks[9].Content);
+        Assert.Equal("food a='b' c = $d", updatedBlocks[9].Content);
+        Assert.Equal(BlockTypes.Code, blocks[9].Type);
+        Assert.Equal(BlockTypes.Code, updatedBlocks[9].Type);
+
         // Arrange
         this._variables.Set("x11", "x11 value");
         this._variables.Set("a", "a value");
         this._variables.Set("_a", "_a value");
+        this._variables.Set("c", "c value");
```

Comment:
What is this doing? Should this be `d` instead?

Reply:
It's setting a default value for c which should get overwritten by the value for `$d` which currently isn't set. I can update the test to include setting `$d` as well. It's not super important since the prompt template engine doesn't render variables for function calls.